http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -499,16 +525,15 @@ if [ ! -z "$JUPYTER_DOCKER_IMAGE" ] ; then
   # For older jupyter images, jupyter_delocalize.py is using 127.0.0.1 as welder's url, which won't work now that we're no longer using `network_mode: host` for GCE VMs
   docker exec $JUPYTER_SERVER_NAME /bin/bash -c "sed -i 's/127.0.0.1/welder/g' /etc/jupyter/custom/jupyter_delocalize.py"
 
-  # Copy gitignore into jupyter container
-
+  # Copy gitignore into jupyter container (ask AOU?)
```

Comment:
@Qi77Qi do you know why we have to copy the gitignore over? I just want to document the use case here